deploy/chart/templates/persistencevolumeclaim.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.mosquitto.persistence.enabled -}}

```

Comment:
This file can be added in the statefulSet as volumeClaimtemplate instead

Reply:
@jlpedrosa I have made this change. Could you please review it?

Reply:
Looks totally fine to me.